console/src/hardware/device/ontology.tsx

```diff
@@ -79,7 +77,7 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
         handleDelete(props);
         break;
       case "link": {
-        const toCopy = `synnax://cluster/${clusterKey}/device/${selection.resources[0].id.key}`;
+        const toCopy = `synnax://cluster/${client.key}/device/${selection.resources[0].id.key}`;
```

Comment:
This won't work. The client key is not equal to the cluster key. Use the cluster key instead.